### ISSUE TITLE
Add Custom Header User-Agent

### ISSUE THREAD

**Issue by ghost** [googlecast/CastReceiver#57]
Hi!
I developed a receiver app and now I need to send a specific request header for decrypting our content. I've already tried this [solution](https://github.com/googlecast/CastReceiver/issues/26), but still not working.

What I can see from my receiver log is the old user-agent, not the new one that I've specified inside licenseRequestHandler, manifestRequestHandler, segmentRequestHandler.

Do you have any idea of how can I achieve this?

**Comment by joeyparrish** [comment]
I don't believe any User Agent (aka browser) allows you to override the User-Agent header in a request.  I expect this is the same for Chromecast as it is for any desktop or mobile browser.
